lib/v2/tju/news/index.js

```diff
@@ -0,0 +1,134 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+const timezone = require('@/utils/timezone');
+
+const news_base_url = 'http://news.tju.edu.cn/';
+const repo_url = 'https://github.com/DIYgod/RSSHub/issues';
+
+const pageType = (href) => {
+    if (href === undefined) {
+        return 'unknown';
+    } else if (!href.startsWith('http')) {
+        return 'in-site';
+    }
+    const url = new URL(href);
+    if (url.hostname === 'news.tju.edu.cn') {
+        return 'tju-news';
+    } else {
+        return 'unknown';
+    }
+};
+
+module.exports = async (ctx) => {
+    const type = ctx.params && ctx.params.type;
+    let path, subtitle;
+
+    switch (type) {
+        case 'focus':
+            subtitle = '聚焦天大';
+            path = 'jjtd.htm';
+            break;
+        case 'general':
+            subtitle = '综合新闻';
+            path = 'zhxw.htm';
+            break;
+        case 'internal':
+            subtitle = '校内新闻';
+            path = 'xnxw1/qb.htm';
+            break;
+        case 'media':
+            subtitle = '媒体报道';
+            path = 'mtbd.htm';
+            break;
+        case 'picture':
+            subtitle = '图说天大';
+            path = 'tstd.htm';
+            break;
+        default:
+            subtitle = '聚焦天大';
+            path = 'jjtd.htm';
+    }
+    let response = null;
+    try {
+        response = await got(news_base_url + path, {
+            headers: {
+                Referer: news_base_url,
+            },
+        });
+    } catch (e) {
+        // ignore error handler
+        // console.log(e);
+    }
+
+    if (response === null) {
+        ctx.state.data = {
+            title: '天津大学新闻网 - ' + subtitle,
+            link: news_base_url + path,
+            description: '链接失效' + news_base_url + path,
+            item: [
+                {
+                    title: '提示信息',
+                    link: repo_url,
+                    description: `<h2>请到<a href=${repo_url}>此处</a>提交Issue</h2>`,
+                },
+            ],
+        };
+    } else {
+        const $ = cheerio.load(response.data);
+
+        let list = null;
```

Comment:
```suggestion
        let list;
```
assignment will be overwritten in the below condition